src/Popover/Popover.js

```diff
@@ -97,9 +100,10 @@ Popover.defaultProps = {
 Popover.propDescriptions = {
     body: 'Node(s) to render in the overlay.',
     control: 'Node to render as the reference element (that the `body` will be placed in relation to).',
+    disableEdgeDetection: 'Set to **true** to render popover without edge detection so popover will not flip from top to bottom with scroll.',
     noArrow: 'Set to **true** to render a popover without an arrow.',
     placement: 'Initial position of the `body` (overlay) related to the `control`.',
-    popperProps: 'Additional props to be spread to the overlay element.',
+    popperProps: 'Additional props to be spread to the overlay element, supported by [popper.js](https://popper.js.org/).',
```

Comment:
I'm not sure if this will work, but can we try adding `{:target="_blank"}` to this link.

Reply:
{:target="_blank"} does not work - and currently can't find a solution to open on a separate page.

Reply:
I just played around with it and if you edit the `_PropertyDescription.js` to `import ReactMarkdown from 'react-markdown/with-html';` and `escapeHtml={false}` to the component, you can change that link to be an anchor tag in regular html with a target blank. See the react-markdown docs for more info!

Reply:
https://github.com/rexxars/react-markdown#parsing-html